__init__.py

```diff
@@ -77,6 +80,21 @@ def mappings_from(nm):
     return answer
 
 
+def button_pressed(hass, button):
+    """Generate HASS bus events for button presses and releases."""
+    payload = {
+        'button': slugify(button.name),
```

Comment:
Should we add the button number (1-5) and the keypad name and keypad_vid as fields, too? I'm trying to think of what we'd want available in a handler and I can imagine each of those things being pretty useful.

Reply:
Yes.  I'll add that (maybe tonight, probably tomorrow).